iroha_client/tests/permissions.rs

```diff
@@ -64,7 +64,7 @@ fn permissions_disallow_asset_transfer() {
     //Then
     assert_eq!(
         rejection_reason,
-        "Can't transfer assets of the other account."
+        "Failed due to low authority: Can\'t transfer assets of the other account."
```

Comment:
I think you need to correct it as the tests will fail you have `Action not permitted...` there now.